packages/server/src/model/serlo.ts

```diff
@@ -956,6 +957,36 @@ export function createSerloModel({
     },
   })
 
+  const moveTaxonomyTerm = createMutation({
+    decoder: DatabaseLayer.getDecoderFor('TaxonomyTermMoveMutation'),
+    mutate: (payload: DatabaseLayer.Payload<'TaxonomyTermMoveMutation'>) => {
+      return DatabaseLayer.makeRequest('TaxonomyTermMoveMutation', payload)
+    },
+    async updateCache({ childrenIds, destination }) {
+      const children = await Promise.all(
+        childrenIds.map((childId) =>
+          getUuidWithCustomDecoder({
+            id: childId,
+            decoder: TaxonomyTermDecoder,
+          })
+        )
+      )
+      const oldParentIds = children.map((child) => child.parentId)
+
+      const idArray = [...childrenIds, ...oldParentIds, destination].filter(
```

Comment:
```suggestion
      const idArray = R.uniq([...childrenIds, ...oldParentIds, destination]).filter(
```

See https://ramdajs.com/docs/#uniq